packages/api/src/routers/svc/rss_feed.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import express from 'express'
+import { readPushSubscription } from '../../datalayer/pubsub'
+import { Subscription } from '../../entity/subscription'
+import { getRepository } from '../../entity/utils'
+import { SubscriptionStatus, SubscriptionType } from '../../generated/graphql'
+import { enqueueRssFeedFetch } from '../../utils/createTask'
+
+export function rssFeedRouter() {
```

Comment:
So this is triggered by the cron and its job is to create one cloud task per RSS subscription, right? Then the rss-handler does the actual fetching of the RSS feeds?

Reply:
yes, exactly